### ISSUE TITLE
Accessibility: Accessible list structure

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/MIPs#205]
## Rationale

Properly structured lists help organize content clearly for all users, especially those using screen readers. When lists use the correct HTML elements (ul, ol, and li), assistive technologies can announce how many items are in the list and make it easier to navigate. If lists are not correctly built, users may not understand the structure or grouping of the content. Clear list markup improves accessibility, readability, and overall user experience.

## 1. Functional Requirements

For each list, it is recommended to use HTML list tags (ul, li or ol,li) as they are the simplest and most reliable method. They work well with all screen readers. As an alternative, ARIA roles (role="list" and role="listitem") may be used if native HTML list tags cannot be applied.

## 2. Examples

- Elements of sidebar menu and sub-menus

![Image](https://github.com/user-attachments/assets/0940b0f7-54aa-4dad-8e46-ab015f36a532)

- List of spaces

![Image](https://github.com/user-attachments/assets/f43548d1-beef-4c32-9fc9-69513c80ae6a)

- List of tasks : to use ol and li
- List of applications in App center
- The list used for the profile elements is malformed 
- The list of note pages

### Access 
   - GUI 
   - API 

### Services & processing
   - Ex: Scheduled Jobs

### Data and persistence
   - Macro data model
   - Persistence layer 
   - Use of caches
## 5. Annexes
   - Reasoning that explain the technical choices
   - Impacts 
   - References
